pennylane/interfaces/jax.py

```diff
@@ -521,7 +524,7 @@ def _execute_fwd_new(
 ):
     """The auxiliary execute function for cases when the user requested
     jacobians to be computed in forward mode (e.g. adjoint) or when no gradient function was
-    provided. This function does not allow multiple derivatives."""
+    provided. This function does not allow multiple derivatives. It does not support shot vectors."""
```

Comment:
Is this a limitation because of adjoint not supporting shot vectors? I think it'd be good to comment on that.

Reply:
The limitation is just that we don't have any device method supporting shot vectors. Potentially we could duplicate the results for device.adjoint_jacobian but I don't see the point. Then it is possible to adapt the jax interface and test it. What do you think?